jupyter-extension/cypress/integration/repo-datum-mode-switch.ts

```diff
@@ -30,9 +30,14 @@ describe('switching between repo and datum mode', () => {
 
     cy.findAllByText('Unload').should('have.length', 2);
     cy.wait(3000);
-    cy.findAllByText('default_images').first().click();
+    cy.get('#jupyterlab-pachyderm-browser-pfs')
+      .findByText('default_images')
+      .dblclick();
     cy.findAllByText('liberty.png').should('have.length', 1);
-    cy.findAllByText('default_images_branch').first().click();
+    cy.get('#jupyterlab-pachyderm-browser-pfs').findByText('/ pfs').click();
```

Comment:
This was really strange I honestly do not know how this test passed before this change. What this test is doing is clicking on a mounted repo folder, checking the file is there, then repeating that for a second mounted repo folder. The problem with the test is that you have to navigate back to the root directory to see the second mounted repo folder, but the test does not do that. Thus I added one step that clicks on the `/ pfs` breadcrumb to make it pass.